NuRadioReco/modules/LOFAR/stationPulseFinder.py

```diff
@@ -275,7 +275,7 @@ def run(self, event, detector):
             station_even_list = []
             station_odd_list = []
             for channel in station.iter_channels():
-                if channel.get_id() == channel.get_group_id():
+                if channel.get_id() == int(channel.get_group_id()[1:]):
```

Comment:
On the lofar/development branch, the channel group ID is already an integer :) So this can be reverted

Reply:
Agreed, will do that :)